_layouts/bib.html

```diff
@@ -2,8 +2,14 @@
 ---
 <!-- _layouts/bib.html -->
       <div class="row">
-        <div class="col-sm-2 abbr">
-        {%- if entry.abbr -%}
+        <div class="col-sm-3 {% if entry.preview %} preview{% else %} abbr{% endif %}">
+        {%- if entry.preview -%}
+          {% if entry.preview contains '://' -%}
+          <img class="preview" src="{{ entry.preview }}">
```

Comment:
let's add some more styling
```suggestion
          <img class="preview z-depth-1 rounded" src="{{ entry.preview }}">
```